tests/functional/monitoring/prometheus/alerts/test_alert_mds_xattr_latency.py

```
@@ -0,0 +1,610 @@
+        nodes: nodes fixture for node operations
+    """
+    log.info("Teardown: Checking if MDS pods are running")
+    mds_pods = cluster.get_mds_pods()
+
+    all_running = True
+    for pd in mds_pods:
+        try:
+            helpers.wait_for_resource_state(
+                resource=pd, state=constants.STATUS_RUNNING, timeout=60
+            )
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)